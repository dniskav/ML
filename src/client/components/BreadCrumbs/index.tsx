import React from 'react';
import styled from 'styled-components';

const Container = styled.div`
  box-sizing: border-box;
  display:flex;
  font-size: 0.85em;
  height: 46px;
  padding: 16px 4px;
  color: #444;
  & a {
    text-decoration: none;
    color: #444;
  }
`;

interface IBreadCrumbs {
  data: any,
}

const BreadCrumbs: React.FC<IBreadCrumbs> = ({ data }) => {

  return(
    <Container>
      Categor√≠a &gt; &nbsp; <a href={data.bUrl}>{data.bName}</a> 
    </Container>
  )
};

export default BreadCrumbs;
