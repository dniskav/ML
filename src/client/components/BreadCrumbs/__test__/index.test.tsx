import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import BreadCrumbs from '../index';

jest.mock('react-router', () => ({
  useParams: () => jest.fn({ id: '1' } as any),
}));

const breadCrumbsProps: any = {
  bName: "",
  bUrl: "",
}

describe('<BreadCrumbs />', () => {
  it("render success", () => {
    const wrapper: ShallowWrapper = shallow(<BreadCrumbs data={breadCrumbsProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
