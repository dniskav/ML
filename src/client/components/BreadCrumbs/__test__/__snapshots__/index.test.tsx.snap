// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BreadCrumbs /> render success 1`] = `ShallowWrapper {}`;
